resources/views/layouts/partials/header/autocomplete.blade.php

```diff
@@ -25,21 +25,16 @@ class="{{ $inputClasses }}"
         >
             <div
                 slot="render"
-                slot-scope="{ downshiftProps: { isOpen }, data: suggestions }"
-            >
-                <ul class="absolute left-2/3 right-auto bg-white border shadow-xl rounded-b-lg lg:rounded-t-lg w-screen sm:w-full lg:w-[960px] xl:ml-0 sm:left-1/2 transform -translate-x-1/2 xl:rounded-t-lg mt-px flex flex-wrap {{ config('rapidez.z-indexes.header-dropdowns') }}" v-if="isOpen && suggestions.length">
+                slot-scope="{ downshiftProps: { isOpen }, data: suggestions }">
+                <ul class="{{ config('rapidez.z-indexes.header-dropdowns') }} absolute left-2/3 right-auto mt-px flex w-screen -translate-x-1/2 transform flex-wrap rounded-b-lg border bg-white shadow-xl sm:left-1/2 sm:w-full lg:w-[960px] lg:rounded-t-lg xl:ml-0 xl:rounded-t-lg" v-if="isOpen && suggestions.length">
                     <li
-                        class="flex w-1/2 sm:w-1/2 md:w-1/3 px-4 my-4"
+                        class="my-4 flex w-1/2 px-4 sm:w-1/2 md:w-1/3"
                         v-for="suggestion in suggestions"
-                        :key="suggestion.source._id"
-                    >
-                        <a :href="suggestion.source.url | url" class="flex flex-wrap w-full h-full" key="suggestion.source._id">
-                            <picture class="contents">
-                                <source :srcset="'/storage/{{ config('rapidez.store') }}/resizes/80x80/magento/catalog/product' + suggestion.source.thumbnail + '.webp'" type="image/webp">
-                                <img :src="'/storage/{{ config('rapidez.store') }}/resizes/80x80/magento/catalog/product' + suggestion.source.thumbnail" class="object-contain lg:w-3/12 self-center" />
-                            </picture>
-                            <div class="px-2 flex flex-wrap flex-grow lg:w-1/2">
-                                <strong class="block hyphens w-full">@{{ suggestion.source.name }}</strong>
+                        :key="suggestion.source._id">
+                        <a :href="suggestion.source.url | url" class="flex h-full w-full flex-wrap" key="suggestion.source._id">
```

Comment:
can you replace the h-full with flex-1? does that work? can you check that for me?